From dd38402eee8a4ff3edcc0c99f437641c66a9ae2b Mon Sep 17 00:00:00 2001
From: Leo Yu-Chi Liang <ycliang@andestech.com>
Date: Fri, 17 Jan 2025 16:35:32 +0800
Subject: [PATCH 170/172] andes: defconfig: Enable IMSIC & APLIC driver (#130)

Enable IMSIC & APLIC support by default to allow choosing
the supported driver by DTS.

CACHE driver is selected by andes Kconfig,
thus the related config is not necessary in defconfig.

Signed-off-by: Leo Yu-Chi Liang <ycliang@andestech.com>
Reviewed-on: https://gitea.andestech.com/RD-SW/opensbi/pulls/130
Reviewed-by: Mina Hui-Min Chou <minachou@andestech.com>
Reviewed-by: Ben Zong-You Xie <ben717@andestech.com>
---
 platform/generic/configs/andes_defconfig | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/platform/generic/configs/andes_defconfig b/platform/generic/configs/andes_defconfig
index f1155f16183c..fdb662048f69 100644
--- a/platform/generic/configs/andes_defconfig
+++ b/platform/generic/configs/andes_defconfig
@@ -1,7 +1,10 @@
 CONFIG_PLATFORM_ANDES_AE350=y
+CONFIG_FDT_CACHE_ANDES_L2C=y
 CONFIG_FDT_IPI=y
 CONFIG_FDT_IPI_PLICSW=y
 CONFIG_FDT_IRQCHIP=y
+CONFIG_FDT_IRQCHIP_APLIC=y
+CONFIG_FDT_IRQCHIP_IMSIC=y
 CONFIG_FDT_IRQCHIP_PLIC=y
 CONFIG_FDT_RESET=y
 CONFIG_FDT_RESET_ATCWDT200=y
@@ -9,5 +12,3 @@ CONFIG_FDT_SERIAL=y
 CONFIG_FDT_SERIAL_UART8250=y
 CONFIG_FDT_TIMER=y
 CONFIG_FDT_TIMER_PLMT=y
-CONFIG_FDT_CACHE=y
-CONFIG_FDT_CACHE_ANDES_L2C=y
-- 
2.34.1

